torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -1096,10 +1096,39 @@ class PyLoweringContext {
     return results;
   }
 
```

Comment:
Could you add another line in the `GetParameterIdTensorMapping` doc comments to the effect of "this will copy the tensor data from device to host"